modules/validatingwebhookconfiguration-yaml-changes.adoc

```diff
@@ -0,0 +1,77 @@
+====
+This section is only applicable if you are using {product-title} version 3.0.41 or newer.
+====
+
+* The webhook is not the part of Sensor anymore, and the reference service changed from Sensor to a dedicated admission control service.
+When you upgrade {product-title} to version 3.0.41 or newer, the upgrade instructions handle this change.
+* With the new configuration, you can enforce security policies on object updates as well, in addition to enforcing on object creation.
+* Beginning for {product-title} version 3.0.55, you can enforce security policies on pod execution and pod port forward events.
+
+[discrete]
+== If Central or Sensor is unavailable
+The admission controller requires an initial configuration from Sensor to work.
+Kubernetes or {ocp} saves this configuration, and it remains accessible even if all admission control service replicas are rescheduled onto other nodes.
+If this initial configuration exists, the admission controller enforces all configured deploy-time policies.
+
+If Sensor or Central becomes unavailable later:
+
+* you will not be able to run image scans, or query information about cached image scans.
+However, admission controller enforcement still functions based on the available information gathered before the timeout expires, even if the gathered information is incomplete.
+* you will not be able to disable the admission controller from the RHACS portal or modify enforcement for an existing policy as the changes will not get propagated to the admission control service.
+
+[NOTE]
+====
+If you need to disable admission control enforcement, you can delete the validating webhook configuration by running the following command:
+
+* On {ocp}:
++
+[source,terminal]
+----
+$ oc delete ValidatingWebhookConfiguration/stackrox
+----
+* On Kubernetes:
++
+[source,terminal]
+----
+$ kubectl delete ValidatingWebhookConfiguration/stackrox
+----
+====
+
+[discrete]
+== Make the admission controller more reliable
+
+Red Hat recommends that you schedule the admission control service on the control plane and not on worker nodes.
+The deployment YAML file includes a soft preference for running on the control plane, however it is not enforced.
+
+By default, the admission control service runs 3 replicas.
+To increase reliability, you could increase the replicas by running the following command:
+* On {ocp}:
++
+[source,terminal]
+----
+$ oc -n stackrox scale deploy/admission-control --replicas=<number_of_replicas>
+----
+* On Kubernetes:
++
+[source,terminal]
+----
+$ kubectl -n stackrox scale deploy/admission-control --replicas=<number_of_replicas>
+----
+
+[discrete]
+== Using with the roxctl CLI
+
+You can use the following options when you generate a sensor deployment YAML file:
+
+* `--admission-controller-listen-on-updates`: If you use this option, {product-title} generates a sensor bundle with a `validatingwebhookconfiguration` pre-configured to receive update events from the Kubernetes or {ocp} API server.
```

Comment:
Should `validatingwebhookconfiguration` be properly cased as `ValidatingWebhookConfiguration`, or just as "validating webhook configuration"?